stable/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 8.14.3
+version: 8.14.4
```

Comment:
Should we bump up the version to `8.15.0`, because we added the new feature and in backward compatible right?

Reply:
Fixed